http-server/src/server.rs

```diff
@@ -389,36 +391,60 @@ impl<M: Middleware> Server<M> {
 
 		let make_service = make_service_fn(move |_| {
 			let methods = methods.clone();
-			let access_control = access_control.clone();
+			let acl = acl.clone();
 			let resources = resources.clone();
 			let middleware = middleware.clone();
 			let health_api = health_api.clone();
 
 			async move {
 				Ok::<_, HyperError>(service_fn(move |request| {
 					let methods = methods.clone();
-					let access_control = access_control.clone();
+					let acl = acl.clone();
 					let resources = resources.clone();
 					let middleware = middleware.clone();
 					let health_api = health_api.clone();
 
 					// Run some validation on the http request, then read the body and try to deserialize it into one of
 					// two cases: a single RPC request or a batch of RPC requests.
 					async move {
-						if let Err(e) = access_control_is_valid(&access_control, &request) {
-							return Ok::<_, HyperError>(e);
+						let host = match http_helpers::read_header_value(request.headers(), "host") {
+							Some(origin) => origin,
+							None => return Ok(malformed()),
```

Comment:
is `malformed()` returning `Response<Body>`?

Reply:
Yes, https://github.com/paritytech/jsonrpsee/blob/master/http-server/src/response.rs#L83-#L89